HTCONDOR-1013 Add example of using AuthTokenSubject (#554)
htcondor/htcondor-ce: docs/v5/configuration/htcondor-routes.md

        SYSTEM_PERIODIC_REMOVE = $(SYSTEM_PERIODIC_REMOVE) || <YOUR REMOVE EXPRESSION>

> To release or put routed jobs on hold if they meet certain criteria, use the `Periodic*` family of attributes.

Reviewer: All these `Periodic` attr related changes seem irrelevant to the ticket. In the future, we should at least split this out into a different commit.

    set_Requirements = original_requirements && ...;
    ```

> ### Setting the accounting group according to the the credential that submitted the job ###

Reviewer: ```suggestion
### Setting the accounting group based on the credential of the submitted job ###
```

Or something along those lines. The credential didn't submit the job but it did come along with the submitted job.

of these attributes cannot be supplied by the *condor_job_router* directly, they will be set based on what credential the *condor_job_router*
uses to submit the routed job.  Because of this the value of these attributes in the routed job is almost never the same as the value in the
original job.  This is different from the way the `x509*` job attributes behaved.

Because of this, the default CE config will copy all attributes that match `AuthToken*` to `orig_AuthToken*` before the route transforms are applied.

Example of setting the accounting group from AuthToken or x509 attributes.

=== "ClassAd Transform"

    ```
    JOB_ROUTER_CLASSAD_USER_MAP_NAMES = $(JOB_ROUTER_CLASSAD_USER_MAP_NAMES) AcctGroupMap
    CLASSAD_USER_MAPFILE_AcctGroupMap = <path-to-mapfile>

    JOB_ROUTER_TRANSFORM_SetAcctGroup @=end
       REQUIREMENTS (orig_AuthTokenSubject ?: x509UserProxySubject) isnt undefined
       EVALSET AcctGroup UserMap("AcctGroupMap", orig_AuthTokenSubject ?: x509UserProxySubject, AcctGroup)
       EVALSET AccountingGroup join(".", AcctGroup, Owner)
    @end

    JOB_ROUTER_PRE_ROUTE_TRANSFORMS = $(JOB_ROUTER_PRE_ROUTE_TRANSFORMS) SetAcctGroup
    ```

> 

Reviewer: Since this example requires a user map file, it'd be helpful to link to the relevant section of the HTCondor manual
